### PR TITLE
Add helm-crd controller

### PR DESCRIPTION
**Pull request by anguslees**
This change adds the helm-crd CRD definition and controller, deployed
as a tiller sidecar.

### CODE REVIEW

**Comment by anguslees** [comment]
(NB: this will trivially conflict with #21, and we should merge that one first)

File: helm-crd.jsonnet [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// CRD and tiller, with controller running as a sidecar
+
+local kube = import "kube.libsonnet";
+local tiller = import "tiller.jsonnet";
```
Comment (prydonius) [inline helm-crd.jsonnet]: should this be `tiller-deployment`?

Comment (prydonius) [reply helm-crd.jsonnet]: actually it doesn't look like this var is being used at all here?

**Review by prydonius — COMMENTED** [review]
I guess this needs updating now that #21 is merged. How do you think we should do this, add it as a sidecar to the Monocular API pod? Or should we run this and another tiller as a separate pod?

File: helm-crd.jsonnet (hunk 1 above)
Comment (anguslees) [reply helm-crd.jsonnet]: oops, yep - copy+paste cruft from upstream helm-crd config.  Removed.

**Comment by anguslees** [comment]
Rebased (as another sidecar on the api pod).  I'm a bit nervous about running multiple tillers, mostly out of ignorance.  Open to any recommendations ...

**Comment by prydonius** [comment]
closing as this is rolled into #36

**Pull request closed without merging**